protocol/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## [Unreleased]
+
+### Features
+
+* [#190](https://github.com/dydxprotocol/v4-chain/pull/190) Added `MsgCreateClobPair` to x/clob to allow creation of new clob pairs.
```

Comment:
nit: should there be subsections for features? for example a group of `feature:gov-updates` PRs should prob be next to each other.

or is this something that we'd do when summarizing the changlogs?

Reply:
I guess there could be if there's a lot of entries that could be grouped together? We can continue without for now and see how many entries end up in a changelog and if there's a lot we can consider breaking it up.